sys-kernel/coreos-kernel/coreos-kernel-4.19.0.ebuild

```diff
@@ -53,8 +53,7 @@ pkg_setup() {
 src_prepare() {
 	# KV_OUT_DIR points to the minimal build tree installed by coreos-modules
 	# Pull in the config and public module signing key
-	# FIXME(bgilbert): remove after final release
-	KV_OUT_DIR=/build/amd64-usr/usr/lib/modules/4.19.0-rc8-coreos/build
+	KV_OUT_DIR="${ROOT%/}/lib/modules/${COREOS_SOURCE_NAME#linux-}/build"
```

Comment:
Apparently, since that variable points at `/usr/src` by default instead of `/lib/modules`, so the `.config` doesn't exist there.  I didn't dig into what changed with it yet; I just pushed that path so tests can run on the new version.